types/level/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for level 6.0
+// Project: https://github.com/Level/level
+// Definitions by: danielfx90 <https://github.com/danielfx90>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { AbstractOptions, ErrorCallback } from 'abstract-leveldown';
+
+import EncodingDown from 'encoding-down';
+
+import { LevelUp } from 'levelup';
+
+export interface LevelDB<K = any, V = any> extends LevelUp<EncodingDown<K, V>> {
+  errors: /* typeof levelerrors */ any; // ? level-errors is not in DT
```

Comment:
this could be just:
```ts
errors: Record<string, Error>;
```
(Liskov substitution, reversed), the list of errors can be an union (`NotFoundError` | `....`). This should be done upstream (levelup) though